test/e2e/e2e.go

```diff
@@ -100,8 +101,11 @@ func RunE2ETests(t *testing.T) {
 			r = append(r, reporters.NewJUnitReporter(path.Join(framework.TestContext.ReportDir, fmt.Sprintf("junit_%v%02d.xml", framework.TestContext.ReportPrefix, config.GinkgoConfig.ParallelNode))))
 		}
 	}
-	klog.Infof("Starting e2e run %q on Ginkgo node %d", framework.RunID, config.GinkgoConfig.ParallelNode)
 
+	// Stream the progress to stdout and optionally a URL accepting progress updates.
+	r = append(r, e2ereporters.NewProgressReporter(framework.TestContext.ProgressReportURL))
```

Comment:
Can we just skip appending the progress reporter if the URL is empty?

Reply:
That means we also don't log to stdout but maybe that's okay?

Reply:
Or maybe logging to stdout should be a separate ginko reporter?

Reply:
I'm not sure I understand the motivation to decouple the URL and stdout printing; all the logic to count/gather/serialize would be the same. Even if we reused this datatype and had one for stdout printing and one for posting to URLs, then the memory footprint and processing overhead would be double (one for each of the reporters).

What is the desired endgoal; you want the ability to disable this functionality?

Reply:
Yes --  it sounds like the new progress reporter is actually doing two things: reporting progress via stdout and reporting progress by sending a POST request to a URL. Based on how ginkgo deals with custom reports, it sounds like they should be decoupled. Reporting to stdout can be on by default but still optional. 

I won't block on this though but something we should consider in the future.